### PR TITLE
Rendering toolkit 05ispc sample addition PR

### PR DESCRIPTION
**Pull request by MichaelRoyceCarroll**
# Adding a New Sample(s)

## Description

Introduce Intel ISPC compilation. ISPC is a new 2022.1 component in Rendering Toolkit.
Discussed in oneapi samples pxt brielfy (20211026 minutes)
Introduced in email thread with @MichaelRoyceCarroll @JoeOster YuryB and VadimS on 20211026.

This sample supports upcoming aims of oneAPI Rendering Toolkit Getting Started Guide. The guide will use this sample to introduce the ISPC component.

Original Source:

- https://github.com/ispc/ispc/tree/main/examples/cpu/simple

Original License:

- The sample is subject to the ISPC license on their github: https://github.com/ispc/ispc/blob/main/LICENSE.txt

## Checklist
Administrative
- [ ] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): <insert Name Here>
- [ ] If you have any new dependencies/binaries, inform the oneAPI Code Samples Project Manager

Code Development
- [ ] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [ ] Adhere to readme template 
- [ ] Enforce format via clang-format config file
- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com

Security and Legal
- [ ] OSPDT Approval (see Project Manager for assistance)
- [ ] Compile using the following compiler flags and fix any warnings, the falgs are: "/Wall -Wformat-security -Werror=format-security"
- [ ] Bandit Scans (Python only)
- [ ] Virus scan

Review
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth(@tomlenth) and/or Project Manager
- [ ] Tested using Dev Cloud when applicable

### CODE REVIEW

**Comment by MichaelRoyceCarroll** [comment]
Looking for help on rebasing this PR properly. It doesn't look right. This should be one commit difference between oneapi-src:development and MichaelRoyceCarroll:RenderingToolkit-05ispc with my fork.... as of 20211209 4:19pm... not 69 commits!

Thanks,
-MichaelC

**Comment by MichaelRoyceCarroll** [comment]
Performed rebase activity per Mike D B Tuckers guidance.

**Review by praveenkk123 — APPROVED** [review]
Approve for CI

**Review by KanclerzPiotr — APPROVED** [review]
Approve for CI

File: RenderingToolkit/GettingStarted/05_ispc_gsg/src/simple.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*
+  Copyright (c) 2010-2011, Intel Corporation
```
Comment (tpyra) [inline RenderingToolkit/GettingStarted/05_ispc_gsg/src/simple.cpp:2]: Is 2011 ok here?
```suggestion
  Copyright (c) 2010-2022, Intel Corporation
```

**Review by tpyra — CHANGES_REQUESTED** [review]

**Review by praveenkk123 — DISMISSED** [review]

File: RenderingToolkit/GettingStarted/05_ispc_gsg/src/simple.ispc [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*
+  Copyright (c) 2010-2011, Intel Corporation
```
[the remark this answers is not in the document]
Comment (MichaelRoyceCarroll) [reply RenderingToolkit/GettingStarted/05_ispc_gsg/src/simple.ispc:2]: Praveen and Tomasz want the year updated. I will apply and resolve. Thanks.

**Review by praveenkk123 — APPROVED** [review]

**Review by praveenkk123 — APPROVED** [review]

**Pull request merged**